pkgs/top-level/python-packages.nix

```diff
@@ -5667,6 +5686,24 @@ in {
     };
   };
 
+  # needed for isso 0.10.6
+  misaka_1 = buildPythonPackage rec {
```

Comment:
This one will need to be added locally to the application `isso`, and not here in `python-packages.nix`.

Reply:
so something like
```nix
let misaka_1 = buildPythonPackage {...}; in
buildPythonPackage {name="isso-..."; propagatedBuildInputs = [misaka_1 ... ] ;...}
```
right?

Reply:
Oh, and what about `html5lib_0_9999999`? Should I use the one in pythonModules? Should that be removed too?

Reply:
You should better use `overridePytonAttrs`: https://github.com/NixOS/nixpkgs/search?q=overridePythonAttrs